docs/UserGuide.md

```diff
@@ -360,169 +373,169 @@ Example: <br>
+* Use the `pin` command for your frequently contacted addresses!<br>
+* For your convenience, if there are repeated fields, only the value of the latest repeated field will be processed. So, if you have mistakenly entered the wrong value, simply retype the field and value in the same line!<br>
 
 </div>
 
 #### Unpinning a contact : `unpin`
 
-Unpins the specified contact on PoochPlanner so that your less important contacts can be removed from the top of your contact list.
+Unpins the specified contact on PoochPlanner so that your less important contacts can be removed from the top of your contacts list.
 
 Format: <br>
 `/unpin ; name : [name]`
 
 Example: <br>
 * `/unpin ; name : Poochie`
 
-  The above command unpins the contact named `Poochie`, provided `Poochie` exists and is pinned in the contact list.
+  The above command unpins the contact named `Poochie`, provided a contact named `Poochie` exists and is pinned in your contacts list.
 
 <br> 
 
 <div style="text-align:center;">
     <br>
     <img src="images/ug-images/command-images/unpincommand.png" alt="Add before" style="width:100%;"/>
-    <medium>Above is an example of unpinning a person contact named Tom :  </medium>
+    <medium>Above is an example of unpinning a person contact named "Tom":</medium>
     <br>
-    <medium>Input: `/unpin ; name : Tom` </medium>
+    <medium>Input: <code>/unpin ; name : Tom</code></medium>
     <br>
-    <medium>Output: Unpins the contact named `Tom` in the contact list. </medium>
+    <medium>Output: Unpins the contact named "Tom" in your contacts list.</medium>
     <br>
 </div>
 
 <br> 
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Constraints:**<br>
+<div markdown="block" class="alert alert-warning">:exclamation: **Constraints:**<br>
 
-* Name is a compulsory field that is case-insensitive but space-sensitive.<br>
+* `name` is a compulsory field that is case-insensitive but space-sensitive.<br>
 
-* Name must be present in PoochPlanner.<br>
+* A contact with the specified `name` must be present in your contacts list.<br>
 
-* Using the unpin command on a contact that has been unpinned does not make any changes to PoochPlanner.<br>
+* Using the `unpin` command on a contact that has been unpinned does not make any changes to your contacts list.<br>
 
-* A contact will still remain unpinned if you use the undo command once on a contact that has been unpinned twice or more.<br>
+* A contact will still remain unpinned if you use the `undo` command once on a contact that has been unpinned twice or more.<br>
 </div>
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tips:**<br>
+<div markdown="block" class="alert alert-primary">:bulb: **Tips:**<br>
 
 * For your convenience, if there are repeated fields, only the value of the latest repeated field will be processed. So, if you have mistakenly entered the wrong value, simply retype the field and value in the same line.<br>
```

Comment:
You can conveniently rectify your mistake by retyping the field and value in your command as only the latest field will be processed.

Reply:
resolved